### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by privat**
Just a refresh on the README content as it is what is shown in github

### CODE REVIEW

**Comment by xymus** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -14,40 +13,37 @@ Some Nit features:
...
-	LICENCE		License of the software
-	misc/		Some additional file for commons text editors and tools
-	tests/		Non-regression test-suite
-	lib/		Nit standard library
-	Makefile	Bootstrap the Nit tools
-	NOTICE		List of the authors
-	README		This file
-	src/		The Nit tool sources (written in Nit)
+ * benchmarks/	Script to bench the compilers
+ * bin/		The Nit tools
```
Comment (Morriar) [inline README.md]: weird indent

**Comment by Morriar** [comment]
+1

File: README.md (hunk 1 above)
Comment (privat) [inline README.md]: It's the `+` from the diff that break tabulations

**Pull request merged**